src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder getSalaryOutput = new StringBuilder();
```

Comment:
The variable name 'getSalaryOutput' is not very informative. Consider renaming it to something like 'salaryReportBuilder' to better reflect its purpose.